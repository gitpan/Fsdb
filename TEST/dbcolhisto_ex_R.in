#fsdb -R C name id test1
name: a
id: 1
test1: 80

name: b
id: 2
test1: 70

name: c
id: 3
test1: 65

name: d
id: 4
test1: 90

name: e
id: 5
test1: 70

name: f
id: 6
test1: 90

#   | dbfilealter -R C
