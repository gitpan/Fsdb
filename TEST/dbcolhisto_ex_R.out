#fsdb low histogram
0 0
10 0
20 0
30 0
40 0
50 0
60 1
70 2
80 1
90 2
#   | dbfilealter -R C
#   | dbcolhisto -S 0 -E 100 -N 10 test1
